### ISSUE TITLE
Incorporate homebridge-indigo2 1.0.0

### ISSUE THREAD

**Issue by waltersobchak40** [Colorado4Wheeler/HomeKit-Bridge#123]
The plugin needs to be updated to incorporate homebridge-indigo2 1.0.0. It appears that this release adds one appealing new feature:
"sendDuplicateUpdates": Send updates to HomeKit Bridge even if value matches cached value (optional, default false)

I've updated to 1.0.0, but it appears that if you edit the config.json to add sendDuplicateUpdates, that change will be removed on startup.

https://github.com/webdeck/homebridge-indigo2

**Comment by Colorado4Wheeler** [comment]
Yes, thank you, I realize the plugin needs updated and it is planned to be updated when I have an opportunity.
